### PR TITLE
Task rung new

### PR DESCRIPTION
**Pull request by vixhha**

### CODE REVIEW

File: Project_FreeRTOS/main/task_rung.h [hunk 1]
```diff
@@ -6,56 +6,50 @@
...
-
-    int rung_value = analogRead(IN_SW_1801P);
+    int rung_currentvalue = analogRead(IN_SW_1801P);
     Serial.print("Rung Value: ");
-    Serial.println(rung_value); 
+    Serial.println(rung_currentvalue); 
    Message sw_send = {0};
       sw_send.id_Tx = OUT_BUZZER;
       sw_send.id_Rx = IN_SW_1801P;
       sw_send.payload = BUZZER_SW_DISABLE;
```
Comment (ThanhtaDEV) [inline Project_FreeRTOS/main/task_rung.h:20]: Please swap values of 2 lines 18 and 19
follow me:
sw_send.id_Tx = IN_SW_1801P;
sw_send.id_Rx = OUT_BUZZER;

File: Project_FreeRTOS/main/task_rung.h [hunk 2]
```diff
@@ -6,56 +6,50 @@
...
       xQueueSend(Platform_Queue, &sw_send, portMAX_DELAY);
```
Comment (ThanhtaDEV) [inline Project_FreeRTOS/main/task_rung.h:21]: remove this line 27

File: Project_FreeRTOS/main/task_rung.h [hunk 3]
```diff
@@ -6,8 +6,14 @@
...
+    int rung_lastValue = 0;
+    int rung_count = 0;
   for(;;)
   {
+    int rung_currentvalue = analogRead(IN_SW_1801P);
+    Serial.print("Rung Value: ");
+    Serial.println(rung_currentvalue); 
    Message sw_send = {0};
       sw_send.id_Tx = OUT_BUZZER;
       sw_send.id_Rx = IN_SW_1801P;
```
Comment (ThanhtaDEV) [inline Project_FreeRTOS/main/task_rung.h:19]: Please swap values of 2 lines 18 and 19
follow me:
sw_send.id_Tx = IN_SW_1801P;
sw_send.id_Rx = OUT_BUZZER;

File: Project_FreeRTOS/main/task_rung.h (hunk 2 above)
Comment (ThanhtaDEV) [inline Project_FreeRTOS/main/task_rung.h]: please remove this line 21 "xQueueSend(Platform_Queue, &sw_send, portMAX_DELAY);"

File: Project_FreeRTOS/main/ha.c
Comment (thaimq) [inline Project_FreeRTOS/main/ha.c:1]: Why do we have this file here?

File: Project_FreeRTOS/main/task_rung.h [hunk 4]
```diff
@@ -6,20 +6,56 @@
...
   for(;;)
   {
+    int rung_currentvalue = analogRead(IN_SW_1801P);
+    Serial.print("Rung Value: ");
+    Serial.println(rung_currentvalue); 
    Message sw_send = {0};
       sw_send.id_Tx = IN_SW_1801P;
       sw_send.id_Rx = OUT_BUZZER;
       sw_send.payload = BUZZER_SW_DISABLE;
       if(xQueueSend(Rung_Queue, &sw_send, portMAX_DELAY) == pdPASS)
```
Comment (thaimq) [inline Project_FreeRTOS/main/task_rung.h:21]: Please remove this line, we have another sending already.

**Review by thaimq — CHANGES_REQUESTED** [review]
Please check my review comments and update new commit ASAP!

**Review by thaimq — APPROVED** [review]
Checked

**Pull request merged**